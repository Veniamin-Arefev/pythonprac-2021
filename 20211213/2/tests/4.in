(rand := __import__("random"),rand.seed("Teskiti testiki, veselo veselo"), [rand.randint(-100, 100) for i in range(48)])[2]