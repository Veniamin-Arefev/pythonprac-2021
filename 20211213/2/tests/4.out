[-97, -96, -93, -87, -85, -83, -78, -74, -73, -73, -68, -61, -50, -50, -30, -30, -27, -22, -14, -8, -8, -7, -7, 0, 6, 7, 10, 15, 20, 30, 31, 33, 40, 41, 44, 48, 48, 49, 58, 60, 62, 62, 64, 68, 77, 87, 92, 92]
